specification/cost-management/resource-manager/Microsoft.CostManagement/stable/2023-03-01/costmanagement.benefits.json

```diff
@@ -314,6 +314,636 @@
           "nextLinkName": "nextLink"
         }
       }
+    },
+    "/providers/Microsoft.Billing/billingAccounts/{billingAccountId}/providers/Microsoft.CostManagement/generateBenefitUtilizationSummariesReport": {
+      "post": {
+        "tags": [
+          "BenefitUtilizationSummariesAsync"
+        ],
+        "operationId": "GenerateBenefitUtilizationSummariesReportAsync_BillingAccountScope",
```

Comment:
>GenerateBenefitUtilizationSummariesReportAsync_BillingAccountScope [](http://example.com/codeflow?start=24&length=66)

See the Linter Warnings: PostOperationIdContainsUrlVerb.

You need to correct the operationId format. For example, here it should be "BillingAccountScope_GenerateBenefitUtilizationSummariesReport". Similarly for other Post Action APIs also.